Makefile

```diff
@@ -1,5 +1,44 @@
 DOT := $(shell command -v dot 2> /dev/null)
 
+RAS_REPOS="ras-party" "ras-secure-message" "ras-frontstage" "ras-collection-instrument" "django-oauth2-test" "ras-rm-auth-service" "respondent-home-ui" "response-operations-ui" "response-operations-social-ui" "sdc-uaa"
+RM_REPOS="rm-sample-service" "rm-case-service" "rm-action-service" "rm-actionexporter-service" "iac-service" "rm-sdx-gateway" "rm-collection-exercise-service" "rm-survey-service" "rm-notify-gateway" "rm-comms-template-service"
+REPOS=${RAS_REPOS} ${RM_REPOS}
+REPOS=${RAS_REPOS} ${RM_REPOS}
+
+check-env:
+ifndef RASRM_HOME
+	$(error RASRM_HOME environment variable is not set.)
+endif
+	@ printf "\n[${YELLOW} RASRM_HOME set to ${RASRM_HOME} ${NO_COLOR}]\n"
```

Comment:
What is `${YELLOW}`?

Reply:
It was to colour the output, I got a bit copy/paste happy from SDX